test/country.mocha.js

```diff
@@ -53,25 +53,19 @@ describe('Country', function() {
                 }
             });
 
-            it('should can find location by domain', function(done) {
-                var data = instance.lookupSync('www.163.com');
+            // Test file doesn't have any IPs that have associated DNS name, travis can't stub hosts
+            it.skip('should can find location by domain', function(done) {
+                var data = instance.lookupSync('shoesfin.NET');
```

Comment:
Curious about this choice of domain vs google.com everywhere else

Reply:
Copied from https://github.com/maxmind/geoip-api-php/blob/master/tests/DomainTest.php before realizing that it didn't work either :(. Travis doesn't have a good way to stub DNS, and I only have a few known IPs within the test files, so I can't come up with a good way to run these tests.